packages/@webex/plugin-meetings/src/reachability/reachability.types.ts

```diff
@@ -57,7 +57,7 @@ export type TransportResultForBackend = {
   clientMediaIPs?: string[];
   latencyInMilliseconds?: string; // amount of time it took to get the first ICE candidate
   reachable?: 'true' | 'false';
-  details?: SubnetDetails[];
+  details?: Array<Omit<SubnetDetails, 'serverIp'> & {serverIps: string}>; // Replace serverIp with serverIps for sending to backend
```

Comment:
this code and comment seems to be incorrect, because it contradicts the type definition of `SubnetDetails` above that already has `serverIps` in it